src/button/tests/functional/Button.ts

```diff
@@ -32,38 +32,36 @@ registerSuite({
 			.end();
 
 	},
-	'button should be disabled'(this: any) {
+	'button should be disabled'() {
 		return getPage(this.remote)
 			.findByCssSelector(`#example-2 .${css.root}`)
 			.isEnabled()
-			.then((enabled: boolean) => {
+			.then(enabled => {
 				assert.isTrue(!enabled, 'The button should be disabled.');
 			})
 			.end();
 	},
-	'button should be toggle-able'(this: any) {
+	'button should be toggle-able'() {
 		return getPage(this.remote)
-			.findByCssSelector(`#example-3 .${css.root}`)
-			.getAttribute('aria-pressed')
-			.then((pressed: string) => {
-				assert.isNull(pressed, 'Initial state should be null');
-			})
-			.click()
-			.then(function (this: any) {
-				// `getAttribute` needs to be placed in then callback, rather than being
-				// directly chainable after `click()` - it won't wait until `click is done`
-				this.getAttribute('aria-pressed')
-					.then((pressed: string) => {
-						assert.strictEqual(pressed, 'true');
-					});
-			})
-			.click()
-			.then(function (this: any) {
-				this.getAttribute('aria-pressed')
-					.then((pressed: string) => {
-						assert.strictEqual(pressed, 'false');
-					});
-			})
+			.findByCssSelector(`#example-4 .${css.root}`)
```

Comment:
I noticed you updated this functional test to target `#example-4` instead of `#example-3`, which like a valid change. Was this passing by accident before and you fixed it intentionally?

Reply:
I'm not really sure why this was passing before since `#example-3` isn't toggle-able. I switched it to `#example-4` which is toggle-able.